endaq/device/schemata/command-response.xml

```diff
@@ -33,6 +33,7 @@
     <!-- These endpoints are kept so that FW can continue to respond to legacy firmware updates and other actions properly, the IDs directly map to ASCII codes-->
     <!-- Note: All 'Legacy' commands except 'LegacyAll' are getting deprecated, and should be avoided. -->
     <MasterElement name="RecStart" id="0x7273" mandatory="0" multiple="0">"rs" Initiate a recording as if button was pressed</MasterElement>
+    <MasterElement name="RecStop" id="0x7275" mandatory="0" multiple="0">"rs" Stop a recording as if button was pressed</MasterElement>
```

Comment:
```suggestion
```
Curious on @StokesMIDE 's thoughts but I think this area is for commands that would go into the command file, which is not the case for the Stop command

Reply:
Good point, I put it for consistency but if the legacy section is for command file stuff, I'll definitely remove it!

Reply:
It doesn't make a functional difference, but since it isn't supported outside of the EBML command element, it should be removed.